src/js/themes/base.d.ts

```diff
@@ -232,6 +232,7 @@ interface ButtonType {
       };
     };
   };
+  style?: Partial<CSSStyleDeclaration>;
```

Comment:
Can you give an example of something that we would want to pass through style that wouldn't be achievable through button props? Just trying to get a better idea of how this could be used